server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaBuilder.java

```diff
@@ -295,6 +298,15 @@ public CentralDogmaBuilder numRepositoryWorkers(int numRepositoryWorkers) {
         return this;
     }
 
+    /**
+     * Sets the maximum allowed age of removed projects and repositories before they are purged.
+     * If unspecified, the default of {@value #DEFAULT_MAX_REMOVED_REPOSITORY_AGE_MILLIS} is used.
```

Comment:
Could you allow a user to disable automatic purge, by specifying zero?

Reply:
That is a nice idea!, It was added [here](https://github.com/line/centraldogma/pull/416/files#diff-e3bb9ea78f820637a7452d39d75bb583R166)